docs/design/add-in-icons-monoline.md

```diff
@@ -0,0 +1,202 @@
+
+#### Background fills
+
+Most icons in the Monoline icon set require background fills. However, there are cases where the object would not naturally have a fill, so no fill should be applied. The following icons have a white fill:
+
+![Five icons have a white fill](../images/monolineicon26.png)
+
+The following icons have no fill. (The gear icon is included to show that the center hole is not filled.)
+![Five icons with no fill](../images/monolineicon27.png)
+
+##### Best practices for fills
+
+###### Dos:
+
+- Fill any element that has a defined boundary, and would naturally have a fill.
+- Use a separate shape to create the background fill.
+- Use **Background Fill** from the [color palette](#color).
+- Maintain the pixel separation between overlapping elements.
+- Fill between multiple objects.
+
+###### Don'ts:
+
+- Don't fill objects that would not naturally be filled; for example, a paperclip.
+- Don't fill brackets.
+- Don't fill behind numbers or alpha characters.
+
+### Color
+
+The color palette has been designed for simplicity and accessibility. It contains 4 neutral colors and two variations for blue, green, yellow, red, and purple. Orange is intentionally not included in the Monoline icon color palette. Each color is intended to be used in specific ways as outlined in this section.
+
+#### Palette
+
+![The four shades of gray in monoline](../images/monoline-grayshades.png)
+
+![The color palette in monoline](../images/monoline-colors.png)
+
+#### How to use color
+
+In the Monoline color pallet, all colors have Standalone, Outline, and Fill variations. Generally, elements are constructed with a fill and a border. The colors are applied in one of the following patterns:
+
+- The Standalone color alone for objects that have no fill.
+- The border uses the Outline color and the fill uses the Fill color.
+- The border uses the Standalone color and the fill uses the Background Fill color.
+
+The following are examples of using color.
+
+![Three icons with color in a border or fill or both](../images/monolineicon28.png)
+
+The most common situation will be to have an element use Dark Gray Standalone with Background Fill.
+
+When using a colored Fill, it should always be with its corresponding Outline color. Example, Blue Fill should only be used with Blue Outline. But there are two exceptions to this general rule:
+
+- Background Fill can be used with any color Standalone.
+- Light Gray Fill can be used with two different Outline colors: Dark Gray or Medium Gray.
+
+#### When to use color
+
+Color should be used to convey the meaning of the icon rather than for embellishment. It should **highlight the action** to the user. When a modifier is added to a base element that has color, the base element is typically turned into Dark gray and Background Fill so that the modifier can be the element of color, such as the case below with the "X" modifier being added to the picture base in the leftmost icon of the following set.
```

Comment:
```suggestion
Color should be used to convey the meaning of the icon rather than for embellishment. It should **highlight the action** to the user. When a modifier is added to a base element that has color, the base element is typically turned into Dark Gray and Background Fill so that the modifier can be the element of color, such as the case below with the "X" modifier being added to the picture base in the leftmost icon of the following set.
```